src/content/en/updates/2018/10/animation-worklet.md

```diff
@@ -0,0 +1,497 @@
+[demo](https://animation-worklet-article.glitch.me/scroller.html), you can
+control the position of the red box.
+
+Note: It might _look_ like you should be able to use `ScrollTimeline` with a
+normal `Animation`, and we agree. This is planned, but currently not supported
+in Chrome.
+
+<video loop muted autoplay
+  src="https://storage.googleapis.com/webfundamentals-assets/animation-worklet/scroller_x264.mp4">
+</video>
+
+If you create a `ScrollTimeline` with an element that doesn't scroll, the
+timeline's `currentTime` will be `NaN`. So especially with responsive design in
+mind, you should always be prepared for `NaN` as your `currentTime`. It’s often
+sensible to default to a value of 0.
+
+Linking animations with scroll position is something that has long been sought,
+but was never really achieved at this level of fidelity (apart from hacky
+workarounds with CSS3D). Animation Worklet allows these effects to be
+implemented in a straightforward way while being highly performant. For example:
+a parallax scrolling effect like this
+[demo](https://animation-worklet-article.glitch.me/parallax.html) shows that it
+now takes just a couple of lines to define a scroll-driven animation.
+
+## Under the hood
+### Worklets
+
+Worklets are JavaScript contexts with an isolated scope and a very small API
+surface. The small API surface allows more aggressive optimization from the
+browser, especially on low-end devices. Additionally, worklets are not bound to
+a specific event loop, but can moved between threads as necessary. This is
+especially important for AnimationWorklet.
+
+### Compositor NSync
+
+You might know that certain CSS properties are fast to animate, while others are
+not. Some properties just need some work on the GPU to be animated, while others
+force the browser to re-layout the entire document. Sites like
+[CSSTriggers.com](https://csstriggers.com/) show you which properties are fast
+to animate, and which are not.
+
+In Chrome (as in many other browsers) we have a process called the compositor,
+whose job it is — and I'm very much simplifying here — to arrange layers and
+textures and then utilize the GPU to update the screen as regularly as possible,
+ideally as fast as the screen can update (typically 60Hz). Depending on which
+CSS properties are being animated, the browser might just need have the
+compositor do it's work, while other properties need to run layout, which is an
+operation that only the main thread can do. Depending on which properties you
+are planning to animate, your animation worklet will either be bound to the main
+thread or run in a separate thread in sync with the compositor.
+
+Note: You should avoid "slow" properties at all costs. Limit yourself to
+animation `opacity` and `transform` to make sure your animations run smoothly
+even on slow devices. Currently, Chrome's implementation doesn't even allow you
+to animate slow properties with AnimationWorklet and will throw a warning at
```

Comment:
We now support slow properties in AnimationWorklet. They just run in sync with main thread. So they still have to be avoided. Here is a [demo](https://aw-playground.glitch.me/width-animation.html) of width animating with a worklet.  

Also we are actually going to remove the warning from devtools. We feel it is a bit spammy and not appropriate for console. We should instead expose this in devtools elsewhere (e.g., in animations panel).  So maybe not mention this.